### ISSUE TITLE
"such as WCAG" ... isn't this a WCAG document.

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag-act#68]
>"Accessibility requirements such as Web Content Accessibility Guidelines, which is specifically designed for web content, can be tested using ACT Rules."

Suggest
>"The Web Content Accessibility Guidelines, which is specifically designed for web content, can be tested using ACT Rules."

**Comment by WilcoFiers** [comment]
@DavidMacDonald The ACT Framework isn't limited to WCAG, as many organisations have accessibility requirements and policies that go beyond that basic requirements of WCAG. We want to make sure that ACT Rules can also be created for those accessibility requirements, as long as those are related to web accessibility, the ACT Framework should be usable.

Obviously WCAG is the primary target for this, and will be the common denominator for most organisations, but we want to ensure that this method scales as well.

Does this address your concerns? If so, please close the issue.

**Comment by DavidMacDonald** [comment]
HOw about
>The Web Content Accessibility Guidelines and other accessibility guideline, which are specifically designed for web content, can be tested using ACT Rules.
